### ISSUE TITLE
slope feature extractor [6]

### ISSUE THREAD

**Issue by cwmeijer** [eEcoLiDAR/eEcoLiDAR#26]
ask Zsofi for the exact explanation

Basically, a slope needs to be fitted through the points and the slope in both the x and y directions need to be returned as feature values (applied only for ground points).

Example: terrain slope: https://apollomapping.com/blog/gdal-calculating-slope

All feature extractors need to implement the feature extractor interface from #19.

**Comment by komazsofi** [comment]
Another example (I am not sure that it is good):
```python
def nv_slope(normals,Z=np.array([0,0,1])):
	ab=normals[0]*Z.T[0]+normals[1]*Z.T[1]+normals[2]*Z.T[2]
ab_length=np.sqrt(normals[0]**2+normals[1]**2+normals[2]**2)*np.sqrt(Z.T[0]**2+Z.T[1]**2+Z.T[2]**2)	
	angle=np.degrees(np.arccos(ab/ab_length))
	return angle
```

**Comment by NicoRenaud** [comment]
I've made the normal plane feature extractor returns both the normal vector and the slope. But we stil can have a sand alone slope extractor if needed

**Comment by komazsofi** [comment]
From my side, it doesn't have to be a separate module. But of course, this is just the user point of view.

**Comment by cwmeijer** [comment]
Implemented in pr #82 .
